frame/dapps-staking/src/pallet/mod.rs

```diff
@@ -114,6 +114,7 @@ pub mod pallet {
 
     /// The current era index.
     #[pallet::storage]
+    #[pallet::whitelist_storage]
```

Comment:
Since we're doing this, please whitelist all necessary storage:
* PalletDisabled
* ForceEra
* NextEraStartingBlock

Reply:
That's true. I updated. thank you for pointing out